lib/arduino_ci/arduino_downloader_windows.rb

```diff
@@ -58,7 +68,11 @@ def extracter
     # Extract the package_file to extracted_file
     # @return [bool] whether successful
     def extract
-      powershell("Expand-Archive", package_file, "-dest", extracted_file)
+      puts 'Extracting ' + package_file + " to " + extracted_file
+      powershell("Expand-Archive", "-Path", package_file, "-DestinationPath", extracted_file)
```

Comment:
Is there a way to mute the output from this command?  It dominates the build log https://ci.appveyor.com/project/ianfixes/arduino-ci/build/1.0.31#L144

Reply:
I couldn't see any flags in the powershell command definition although it suggests there might be common parameters. Standard suggestions on google appear to be to pipe to null. I just pushed up a version with this but it looks like you have merged already so I am not sure if that was included or not.

Note - This doesn't happen when run locally, you just get a nice nice progress bar:
` [oooooooooooo........................................ ]`

I did wonder if both extract and move could be done using a ruby dependency rather than powershell, but my ruby knowledge is minimal to none.

Reply:
Putting those in pure ruby is probably the right way to go.  I more or less ported the `install.sh` script from Adafruit to boostrap this project... so now that it looks like I'm headed for cross-platform fun (beyond my most ambitious expectations, honestly, in a good way!), I will plan to take a closer look at that.

Of course, having working cross-platform CI is a great enabler for that kind of thing in the first place... so I can't thank you enough for your help here.

Reply:
Also, I cherry-picked your commit with no problems.  Thanks